src/sentry/static/sentry/app/views/settings/account/avatar.jsx

```diff
@@ -1,45 +1,55 @@
 import PropTypes from 'prop-types';
 import React from 'react';
-import styled from 'react-emotion';
 import createReactClass from 'create-react-class';
+import styled from 'react-emotion';
 
-import Panel from '../components/panel';
-import PanelBody from '../components/panelBody';
-import PanelHeader from '../components/panelHeader';
-import AlertActions from '../../../actions/alertActions';
+import {
+  addErrorMessage,
+  addSuccessMessage,
+} from '../../../actionCreators/settingsIndicator';
+import {t} from '../../../locale';
 import ApiMixin from '../../../mixins/apiMixin';
+import Avatar from '../../../components/avatar';
 import AvatarCropper from '../../../components/avatarCropper';
-import RadioGroup from '../components/forms/radioGroup';
 import LoadingError from '../../../components/loadingError';
 import LoadingIndicator from '../../../components/loadingIndicator';
-import {t} from '../../../locale';
+import Panel from '../components/panel';
+import PanelBody from '../components/panelBody';
+import PanelHeader from '../components/panelHeader';
+import RadioGroup from '../components/forms/radioGroup';
+import SentryTypes from '../../../proptypes';
 
-const AvatarSettings = createReactClass({
-  displayName: 'AvatarSettings',
+const AccountAvatar = createReactClass({
+  displayName: 'AccountAvatar',
 
   propTypes: {
     userId: PropTypes.number,
+    user: SentryTypes.User,
+    onSave: PropTypes.func,
   },
 
   mixins: [ApiMixin],
 
+  getDefaultProps() {
+    return {
```

Comment:
when would this default be used?

Reply:
It's to be a bit more defensive since we call `onSave` in a success handler.